rkyv/src/util/aligned_vec.rs

```diff
@@ -863,7 +893,7 @@ impl io::Write for AlignedVec {
 }
 
 // SAFETY: AlignedVec is safe to send to another thread
```

Comment:
Nit: update these comments to `AlignedByteVec`. There's also a fair bit of documentation that refers to `AlignedVec` that should now refer to `AlignedByteVec`.

Reply:
Yes, this PR is a very rough draft at present. I wasn't at all sure if you'd think it was a good idea.

Reply:
I think it moves rkyv in a positive direction. 🙂

Reply:
I think this is the only thing blocking merging right now.